dialogue-core/src/main/java/com/palantir/dialogue/core/RetryingChannel.java

```diff
@@ -80,6 +83,7 @@
             ClientConfiguration.RetryOnTimeout retryOnTimeout) {
         this(
                 delegate,
+                new DefaultTaggedMetricRegistry(),
```

Comment:
I think we need to add this to the constructor and pass the registry from our ClientConfiguration

Reply:
Yes we do do that, this constructor is only visible for testing. Let me  make that more clear

Reply:
oh doh, I misread, sorry!